File: CrEST/Microsoft.Crest.Web.API/CrEST.Models/SowMetadata.cs
using CrEST.Data.Models;
using System.Collections.Generic;

namespace CrEST.Models
{
    public class SowMetadata
    {
        public List<Supplier> Suppliers { get; set; }
        public List<Itorg> ItOrg { get; set; }
        public List<int> ContractIds { get; set; }
        public List<int> CompanyCodes { get; set; }
    }
}

File: CrEST/Microsoft.CrEST.UI/Microsoft.CrEST.UI/angularCrest/app/sla/shared/models/sowMetaData.ts
﻿export class SowMetaData {
    public contractIds: Array<number>;
    public suppliers: Array<Supplier>;
    public itOrg: Array<ITOrg>;
    public serviceClass: Array<ServiceClass>;
    public companyCodes: Array<number>;
};

export class Supplier {
    public supplierId: number;
    public supplierName: string;
    public isCrest: boolean;
}


export class ITOrg {
    public itOrgId: number;
    public itOrgName: string;

}
export class ServiceClass {
    public id: number;
    public serviceClassName: string;
}
